packages/toolkit/.changes/next-release/Feature-dd85e352-be76-4b69-8225-03b4e784ca10.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Lambda: add progress status item for Lambda invoke actions"
```

Comment:
I wonder if we should be more explicit and indicate that it's both local invoke and remote invoke.